config-tests/test_remote_settings.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+
+from kinto_http import Client
+from kinto_http.patch_type import JSONPatch
+
+
+@pytest.mark.settings
+@pytest.mark.skipif(
+    pytest.config.getoption("env") != "stage",
+    reason="Test only runs on staging"
+)
+def test_create_collection_in_main_bucket(conf, env):
+    # Create a record inside the 'main-workspace' bucket and then
+    # request that a specific user review it
+    client = Client(
+        server_url=conf.get(env, "writer_server"),
+        auth=(os.getenv("RS_USER_1_LOGIN"), os.getenv("RS_USER_1_PASSWD"))
+    )
+    data = {"qa": "testing"}
+    client.create_collection(id="qatest", if_not_exists=True, bucket="main-workspace")
+    cr_response = client.create_record(data=data, collection="qatest", bucket="main-workspace")
+    new_record_id = cr_response["data"]["id"]
+    changes = JSONPatch(
+        [dict(op="add", path="/data/members/0", value="kinto_rs_2")]
+    )
+    client.patch_group(
+        id="qatest-reviewers",
+        bucket="main-workspace",
+        changes=changes
+    )
```

Comment:
Since you reuse the same `qatest` collection for each run, the `qatest-reviewers` might already have `kinto_rs_2` in its members.

I don't know the [JSON Patch spec](https://tools.ietf.org/html/rfc6902) in details, but there might be a way to express `add this value to the array if not there yet`.

Reply:
I was actually thinking of changing the test to make the collection name random to avoid this sort of thing